src/styles/Constants.ts

```diff
@@ -26,4 +26,7 @@ export enum Colours {
   TriageBlack = '#000000',
   TriageSelectedBlack = '#BFBFBF',
   SnackbarButtonBlue = '#28B2FF',
+  Marker = '#EE502E',
+  MarkerNotClicked = '#F7B8AB',
```

Comment:
nitpick
```suggestion
  MarkerNotSelected = '#F7B8AB',
```

Reply:
I will include this in my cleanup ticket! Thanks for the catch.